### PR TITLE
Add significant_text challenge

### PR DESCRIPTION
**Pull request by nik9000**
I've been looking into a performance drop in significant text in 7.9 and
needed a rally challenge to test against. This is that!

### CODE REVIEW

File: geonames/operations/default.json [hunk 1]
```diff
@@ -348,4 +348,94 @@
...
+              "sig": {
+                "significant_text": {
+                  "field": "alternatenames"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
```
Comment (danielmitterdorfer) [inline geonames/operations/default.json:441]: Nit: Missing new line at the end of the file.

**Review by danielmitterdorfer — APPROVED** [review]
Thanks for the PR! One nit but LGTM.

**Comment by nik9000** [comment]
Thanks @danielmitterdorfer !

**Comment by nik9000** [comment]
Thanks @danielmitterdorfer !

**Pull request merged**